packages/nextjs-template/utils/validation.js

```diff
@@ -0,0 +1,18 @@
+const fs = require("fs-extra");
+const path = require("path");
+const { parse, treeMenuSchema } = require("@dendronhq/common-all");
+const { env } = require('./env')
+
+const dataDir = env.DATA_DIR
+if (!dataDir) {
+  throw new Error("DATA_DIR not set");
+}
+
+// check if `tree.json` is formated correctly
+const treeInput = fs.readJSONSync(path.join(dataDir, "tree.json"));
+const treeMenuResp = parse(treeMenuSchema, treeInput);
+if (treeMenuResp.error) {
+  throw new Error(
```

Comment:
don't forget to close the parenthesis `(where your dendron.yml file is located)`.

Reply:
[`3385ddd` (#3542)](https://github.com/dendronhq/dendron/pull/3542/commits/3385ddd52a4efec7b0c3c8cdc342bb11f46d4eef)